gem/templates/rwanda/base.html

```diff
@@ -175,12 +174,15 @@
                 <h1 class="nav-list__title">{% trans "Featured" %}:</h1>
                 <ul class="nav-list-inner">
                   {% for section_page in sections %}
-                    {% if section_page.show_in_menus == True %}
-                      <li class="nav-list-inner__item">
-                        <a href="{% pageurl section_page.specific %}" class="nav-list__anchor">
-                          {{section_page.title}}
-                        </a>
-                      </li>
+                    {% content_is section_page "Events" as is_event %}
```

Comment:
Does this handle translations? I presume the `sections` list that we're looping through is the translated sections.